domain/src/main/java/com/nextstep/edu/domain/Expression.kt

```diff
@@ -0,0 +1,58 @@
+package com.nextstep.edu.domain
+
+import java.util.Optional
+
+class Expression(
+    operands: String = ""
+) {
+
+    private val calculator = Calculator
+    private var operands: String = ""
```

Comment:
```suggestion
    var operands: String = ""
        private set
```

코틀린의 멤버변수는 프로퍼티의 특성을 가지고 있습니다!
이미 getter를 가지고 있기 때문에 getter를 추가 정의하지 않는 것이 더욱 바람직합니다.

Reply:
추가적으로 operands는 피연산자의 의미를 가지고있는데요~
연산자도 같이 포함될 수 있는 식이기때문에 변수명은 변경하는 것을 추천드립니다.

Reply:
아하!! 그러네요!! 방어적 연산자로 열어두면 더 깔끔해지겠네요~ 해당 내용 수정하도록 하겠습니다ㅎㅎ